File: app/src/main/java/com/manoj/dlt/utils/TextChangedListener.java
package com.manoj.dlt.utils;

import android.text.Editable;
import android.text.TextWatcher;

public abstract class TextChangedListener implements TextWatcher
{

    @Override
    public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) { }

    @Override
    public void afterTextChanged(Editable editable) { }
}

File: app/src/main/kotlin/com/manoj/dlt/utils/TextChangedListener.kt
package com.manoj.dlt.utils

import android.text.Editable
import android.text.TextWatcher

abstract class TextChangedListener : TextWatcher {

    override fun beforeTextChanged(charSequence: CharSequence, i: Int, i1: Int, i2: Int) {
    }

    override fun afterTextChanged(editable: Editable) {
    }
}
